authortechnical.Rmd

```diff
@@ -391,7 +391,32 @@ Value 21 | Value 22       | Value 23
 Value 31 | Value 32       | Value 33
 {.table .table-responsive .table-striped}
 ```
+  
+### Math
+  
+If you want to use mathematical formula,
+  
+* install the package [katex](https://docs.ropensci.org/katex/).
+* add `math: true` to your post YAML.
+* for all math you'll need to double the backslashes. All "\" become "\\".
+* for inline math (`$1+1$`), use no dollar sign but instead.
+  
+```r
+`r katex::katex_html("1 + 1", displayMode = FALSE)`
```

Comment:
To be clear, they're supposed to use the function, right? So should this just be presented as un-evaluated code?
That is to say use ` ``` `, and no `r` ? The preview looks like html code to me, but changing it to cat looks better.
```suggestion
cat("`r katex::katex_html(\"1 + 1\", displayMode = FALSE)`")
```